docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -1842,6 +1848,31 @@ Comma-separated list of attributes to include in all new spans, such as `key1:va
 
 Can be set with the environment variable `OTEL_RESOURCE_ATTRIBUTES` (use `=` instead of `:` with the environment variable).
 
+### sampler_type
+
+Default value is `const`.
+
+Specifies the type of sampler: `const`, `probabilistic`, `ratelimiting`, or `remote`.
+
+### sampler_param
+
+Default value is `1`.
+
+Sampler configuration parameter. Depending on the value of `sampler_type`, it can be `0`, `1`, or a decimal value in between.
+
+- For `const` sampler, `0` or `1` to always `false`/`true` respectively
```

Comment:
```suggestion
- For `const` sampler, `0` or `1` means `false`/`true` respectively
```

Reply:
Does that suggestion  ^ make sense?

Reply:
yep - the "always" wording was perhaps trying to be too succinct. What was intended is that the sampling decision would always be `false` or `true` depending on the value.

There's probably a better way to word it. How about:

```suggestion
- For `const` sampler, use `0` to never sample or `1` to always sample
```

Reply:
👍 , that works for me, thank you.